src/components/AccountSettings.tsx

```diff
@@ -1,330 +1,277 @@
+            </FormGroup>
+            <FormGroup className='required'>
+              <FormLabel>Email address</FormLabel>
+              <FormControl onChange={e => setPrimaryEmail(e.target.value)} value={primaryEmail}/>
+            </FormGroup>
+            <FormGroup>
+              <FormLabel>First name</FormLabel>
+              <FormControl onChange={e => setFirstName(e.target.value)} value={firstName}/>
+            </FormGroup>
+            <FormGroup>
+              <FormLabel>Last name</FormLabel>
+              <FormControl onChange={e => setLastName(e.target.value)} value={lastName}/>
+            </FormGroup>
+            <FormGroup>
+              <FormLabel>Current position</FormLabel>
+              <FormControl placeholder='e.g Principal Investigator' onChange={e => setPosition(e.target.value)} value={position}/>
+            </FormGroup>
+            <FormGroup>
+              <FormLabel>Current affiliation</FormLabel>
+              <FormControl placeholder='e.g. Example University' onChange={e => setCompany(e.target.value)} value={company}/>
+            </FormGroup>
+            <FormGroup>
+              <FormLabel>Industry</FormLabel>
+              <FormControl onChange={e => setIndustry(e.target.value)} value={industry}/>
+            </FormGroup>
+            <FormGroup>
+              <FormLabel>Website</FormLabel>
+              <FormControl placeholder='https://example.com' onChange={e => setUrl(e.target.value)} value={url}/>
+            </FormGroup>
+            <FormGroup>
+              <FormLabel>City, Country</FormLabel>
+              <FormControl onChange={e => setLocation(e.target.value)} value={location}/>
+            </FormGroup>
+            <FormGroup>
+              <FormLabel>Institutional affiliation</FormLabel>
+              <FormControl/>
+            </FormGroup>
+            <Button onClick={updateUserProfile} disabled={!changeInForm} variant='contained'>Save Changes</Button>
+          </Form>
+        </div>
+        <div ref={changePasswordRef} className='account-setting-panel main-panel'>
+          <h3>Change Password</h3>
+          <ChangePassword/>
+        </div>
+        <div ref={trustCredentialRef} className='account-setting-panel main-panel'>
+          <h3 >Trust & Credentials</h3>
+          <p>This section lists the various ways we support verifying your trust and identity in order to permit access to our products. You may be asked to complete any of the following as part of requesting access within a system.</p>
+          <div className='credential-partition'>
+            <h4>Terms and Conditions for Use</h4>
+            <i>Required to register</i>
+            <p>You must affirm your agreement to follow these terms and conditions in order to create an account. </p>
+            <div className='item-completion'>
+              {/* Check if user finished Terms of Use */}
             </div>
```

Comment:
Needs to check if user has completed the Terms of Use

Reply:
Most calls respond with an error if Synapse ToU are not agreed to.